### DISCUSSION TITLE
Filtering GitHub issues by project column and other meta in Poker

### DISCUSSION THREAD

**Discussion by ackernaut** [ParabolInc/parabol#6163] [Feedback]
[This company](https://www.notion.so/parabol/Feedback-from-atsign-com-The-Company-d9faab4e89e7444cb4447b54c325f601) 🔒  makes a lot of use of their project board in GitHub (see the screenshot in the doc, or watch the Loom), and [wrote a script to be able to export issues](https://github.com/atsign-company/dump_cards) to include in [planningpoker.com](https://www.planningpoker.com/).

They had a few questions:
>Can the tool handle importing dimensions from our existing GitHub beta board?

and

>Can creation date and last date of activity for each card be identified and exported to add an ‘issue age’ dimension to our GitHub view? (View of our board attached)

My question is: what meta is available on issues and project that could be used in our Scope phase so that they can get the right issues into their poker session?

**Comment by mattkrick** [comment]
> Can the tool handle importing dimensions from our existing GitHub beta board?

GitHub's API for project beta is still a long ways out until it's useful. that's where we're headed though.

> Can creation date and last date of activity for each card be identified and exported to add an ‘issue age’ dimension to our GitHub view? (View of our board attached)

I'm not sure I understand this question, I think they may want to dive into github actions. both those fields come from github, right?

> My question is: what meta is available on issues and project that could be used in our Scope phase so that they can get the right issues into their poker session?

not much now, we'll need projects beta to progress a little more
